2a_model/src/models/Snakefile_base.smk

```diff
@@ -27,6 +27,46 @@ rule as_run_config:
         asRunConfig(config,output[0])
 
 
+rule prep_io_data:
```

Comment:
I put this back in the Snakefile. I had originally thought of not mixing Snakemake and Targets which is why I had this in Targets before (so we didn't have to call Snakemake to produce it). Since then, I realized it was going to be too tricky to try to circumvent Snakemake altogether. Since we are calling Snakemake anyway here (https://github.com/USGS-R/drb-do-ml/blob/c495411c579dd76c78dabc0c346893c2e17b249f/2a_model.R#L152), I figured it made sense to have this python function call back in snakemake.